docs/components/password/index.html

```diff
@@ -0,0 +1,143 @@
+        Showing and hiding password text.
+    </p>
+
+    <div class="example" id="vue-passwordInputDemo" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <hx-password-control>
+            <input
+              id="pwdLabel"
+              type="password"
+              v-model="label"
+              minlength="8"
+              placeholder="Please Enter Password"
+              pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+              required
+            />
+            <label for="pwdLabel">
+              Label Text
+            </label>
+          </hx-password-control>
+
+          <fieldset>
+            <legend class="beta-hxFieldName">Label Annotation</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkHasAsterisk"
+                type="checkbox"
+                v-model="hasAsterisk"
+              />
+              <label for="chkHasAsterisk">
+                <hx-checkbox></hx-checkbox>
+                Required
+              </label>
+            </hx-checkbox-control>
+
+            <hx-checkbox-control>
+              <input
+                id="chkHasOptional"
+                type="checkbox"
+                v-model="hasOptional"
+              />
+              <label for="chkHasOptional">
+                <hx-checkbox></hx-checkbox>
+                Optional
+              </label>
+            </hx-checkbox-control>
+          </fieldset>
+
+          <fieldset>
+            <legend class="beta-hxFieldName">Control Options</legend>
+            <hx-checkbox-control>
+              <input
+                id="chkDisabled"
+                type="checkbox"
+                v-model="isDisabled"
+              />
+              <label for="chkDisabled">
+                <hx-checkbox></hx-checkbox>
+                Disabled
+              </label>
+            </hx-checkbox-control>
+
+            <hx-checkbox-control>
+              <input
+                id="chkRequired"
+                type="checkbox"
+                v-model="isRequired"
+              />
+              <label for="chkRequired">
+                <hx-checkbox></hx-checkbox>
+                Required
+              </label>
+              <p>
+                (Invalid styling applied <i>after</i> user interaction.)
+              </p>
+            </hx-checkbox-control>
+          </fieldset>
+        </form>
+      </header>
+
+      <div>
+        <hx-password-control>
+          <input
+            id="pwdInputDemo"
+            type="password"
+            :disabled="isDisabled"
+            :required="isRequired"
+            v-model="label"
+            pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"
+          />
+          <label
+            for="pwdInputDemo"
+            :class="{ hxOptional: hasOptional, hxRequired: hasAsterisk }"
+          >
+            Password
```

Comment:
Entering the password should not change the label value:

<img width="709" alt="Screen Shot 2020-02-26 at 1 49 26 PM" src="https://user-images.githubusercontent.com/10120600/75382243-bbaddf00-589f-11ea-81f8-3c86ca7262ea.png">

Reply:
Changes have been done.